docs/reference/cli.md

```diff
@@ -24,33 +24,15 @@ Before using Quickwit with an object storage, check out our [advice](../administ
 
 `quickwit <command name> --help` displays the documentation for the command and a usage example.
 
-## Environment Variables
```

Comment:
In the last review you asked to put the QW_CONFIG env at the top, since it simpliyfies the whole usage. I think that makes sense

Reply:
Yes indeed, but I only wanted to talk about `QW_CONFIG` which is the most important and leave other env variables at the end.